WooCommercePOS/WooCommercePOS/Presentation/PointOfSaleEntryPointView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+public struct PointOfSaleEntryPointView: View {
+    // TODO:
+    // Temporary. DI proper product models once we have a data layer
+    private let viewModel: PointOfSaleDashboardViewModel = {
+        let products = ProductFactory.makeFakeProducts()
+        let viewModel = PointOfSaleDashboardViewModel(products: products)
+
+        return viewModel
+    }()
+
+    private var shouldHideWooAppTabBar: ((Bool) -> Void)? = nil
```

Comment:
nit: I think this can be a let. "should*" also sounds like a boolean, maybe it can match the parameter name
```suggestion
    private var hideAppTabBar: ((Bool) -> Void)? = nil
```

Reply:
Done! https://github.com/woocommerce/woocommerce-ios/pull/12700/commits/27f8499b3be1c89df547ff94f656e9804309d9fc